tests/func/test_update.py

```diff
@@ -172,3 +172,35 @@ def test_update_import_url(tmp_dir, dvc, tmp_path_factory):
 
     assert dst.is_file()
     assert dst.read_text() == "updated file content"
+
+
+def test_update_git_tracked(tmp_dir, dvc, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.scm.repo.index.remove([".dvc"], r=True)
+        shutil.rmtree(".dvc")
+        erepo_dir.scm_gen("file", "first version")
```

Comment:
you could do:
```suggestion
        erepo_dir.scm_gen("file", "first version", commit="first version")
```

Reply:
I can do that but would still need a commit to remove `.dvc` in a way that can be cloned as a regular git repo

Reply:
Can you also simplify this?
```suggestion
        erepo_dir.scm_gen("file", "first version", commit="first version")
```

Reply:
Now done through `git_dir` fixture